src/DIRAC/Core/Utilities/ElasticSearchDB.py

```diff
@@ -125,6 +126,8 @@ def __init__(
         :param str client_cert: Client certificate.
         """
 
+        password = urlparse.quote_plus(password)
```

Comment:
Have up you checked if this is correct for open[elastic]search?

Reply:
I have read it's correct (did not try to verify).